src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -2096,44 +2095,43 @@ class MarkFromRootsTask : public AbstractGangTask {
   }
 };
 
-class PCRefProcTask : public AbstractGangTask {
-  typedef AbstractRefProcTaskExecutor::ProcessTask ProcessTask;
-  ProcessTask& _task;
-  uint _ergo_workers;
-  TaskTerminator _terminator;
+class PCRefProcClosureContext : public AbstractClosureContext {
```

Comment:
The abbreviation used for Parallel Compact is `ParallelCompact` or `ParCompact[ion]` or `PSParallelCompact` otherwise afaics. Please do not add even more of those (yeah, there is already `PCAddThreadRootsMarkingTaskClosure`)...

Reply:
I will rename PC -> ParallelCompact. I will ignore the historical roots of the "mirror" class names PCRefProcTask, RefProcTaskExecutor.

Reply:
I will also rename PSRefProcClosureContext -> ParallelScavengeRefProcClosureContext